app/team.go

```diff
@@ -675,11 +675,11 @@ func (a *App) GetTeamMembersForUser(userId string) ([]*model.TeamMember, *model.
 }
 
 func (a *App) GetTeamMembersForUserWithPagination(userId string, page, perPage int) ([]*model.TeamMember, *model.AppError) {
-	result := <-a.Srv.Store.Team().GetTeamsForUserWithPagination(userId, page, perPage)
```

Comment:
You can just `return a.Srv.Store.Team().GetTeamsForUserWithPagination(userId, page, perPage)` here.